internal/controller/linodevpc_controller.go

```diff
@@ -153,6 +154,12 @@ func (r *LinodeVPCReconciler) reconcile(
 		return res, err
 	}
 
+	// Pause
+	// We don't have much to do, but simply requeue without an error if we are paused or if we were recently unpaused
+	if isPaused, conditionChanged, err := paused.EnsurePausedCondition(ctx, vpcScope.Client, nil, vpcScope.LinodeVPC); err != nil || isPaused || conditionChanged {
```

Comment:
can we use the isPaused predicate for these child resources? https://github.com/kubernetes-sigs/cluster-api/blob/main/util/predicates/generic_predicates.go#L217-L227